build/Targets/VSL.Versions.targets

```diff
@@ -1,9 +1,17 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- This controls the version numbers of the build that we are producing -->
   <PropertyGroup>
-    <RoslynSemanticVersion Condition="'$(RoslynSemanticVersion)' == '' AND '$(UseVisualStudioVersion)' == 'true'">15.0.0</RoslynSemanticVersion>
-    <RoslynSemanticVersion Condition="'$(RoslynSemanticVersion)' == ''">2.0.0</RoslynSemanticVersion>
+    <!-- This the assembly version of Roslyn from the .NET assembly perspective. It should only be revved during significant point releases. -->
+    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.0.0</RoslynAssemblyVersionBase>
+    <!-- This is the file version of Roslyn, as placed in the PE header. It should be revved during point releases, and is also what provides the basis for our NuGet package versioning. -->
+    <RoslynFileVersionBase Condition="'$(RoslynFileVersionBase)' == ''">2.0.0</RoslynFileVersionBase>
```

Comment:
Just to confirm...

AssemblyVersionBase will rev during Major/Minor releases (2.0.0.0 in RTM, 2.1.0.0 in Update 1, etc...)

RoslynFileVersionBase will rev during Major/Minor/Patch releases (2.0.0.0 in RTM, 2.0.1.0 in RTM MicroUpdate, etc).